src/convertToObject.js

```diff
@@ -12,7 +12,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map(item => item.split(':')
+      .map(line => line.trim()))
```

Comment:
```suggestion
    .map(line => line.trim()))
```

Reply:
that map is called on item.split(':'). should we not indent it in this case?